tests/Aspire.Hosting.Azure.Tests/AzureBicepProvisionerTests.cs

```diff
@@ -106,14 +106,12 @@ public async Task ResourceWithTheSameBicepTemplateAndParametersHaveTheSameCheckS
     [InlineData("alpha 123")]
     public void WithParameterDoesNotAllowParameterNamesWhichAreInvalidBicepIdentifiers(string bicepParameterName)
     {
-        var ex = Assert.Throws<ArgumentException>(() =>
+        Assert.Throws<ArgumentException>(() =>
         {
             using var builder = TestDistributedApplicationBuilder.Create();
             builder.AddAzureConstruct("construct", _ => { })
                    .WithParameter(bicepParameterName);
         });
-
-        Assert.Equal("Bicep identifiers must only contain alpha, numeric, and _ characters and must start with an alpha or _ character. (Parameter 'name')", ex.Message);
```

Comment:
The exception message comes from Azure.Provisioning now, so I didn't feel it was necesary for us to validate it. The exception message is different depending on empty, starts with number, etc.